### PR TITLE
releaseNotes_mar14th-mar20th

### PR DESCRIPTION
**Pull request by qpetraroia**

### CODE REVIEW

**Review by weinong — APPROVED** [review]
LGTM

**Review by miwithro — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-03-22
+
+This release is rolling out to all regions - ETA for conclusion 2021-03-31 for public cloud.
+
+### Announcements
+
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Kubernetes version 1.17 will be deprecated in the last week of March 2021.
```
Comment (palma21) [inline CHANGELOG.md:10]: I'd refer to next week now. "Next week starting on ..."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+* CSI Drivers will become default for [Kubernetes versions 1.21+](https://docs.microsoft.com/azure/aks/csi-storage-drivers).
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* For all AKS clusters using kubernetes v1.20+, coredns will be upgraded to version 1.8.3. This will remove resyncperiodand upstream from the Kubernetes plugin.
```
Comment (palma21) [inline CHANGELOG.md:14]: Capitalize CoreDNS

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+
+### Release Notes
+
+* Bug Fixes
+  * Fixed an issue regarding indecisiveness in Kubernetes versions and the auto-upgrade feature in ARM templates. Read more [here](https://github.com/Azure/AKS/issues/2138).
```
Comment (palma21) [inline CHANGELOG.md:19]: a) Good job capturing the issue.
b) Make sure we update them when included (did this now for this one, check it out)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
...
+* Component Updates
+  * OSM addon has been updated to version 0.8.
```
Comment (palma21) [inline CHANGELOG.md:21]: This is not released yet, please remove.

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (qpetraroia) [reply CHANGELOG.md:19]: Noted. Will remember this for next time.

**Review by palma21 — APPROVED** [review]

**Pull request merged**